python-threatexchange/threatexchange/content_type/photo.py

```diff
@@ -19,3 +21,75 @@ class PhotoContent(ContentType):
       * frames from videos
       * thumbnails of videos
     """
+    @classmethod
+    def rotate_image(cls, image_data: bytes, angle: float) -> bytes: 
+        """
+        Rotate an image by a given angle 
```

Comment:
Include a note on why we might want to do this :P 

Namely, this is help for testing hash algorithms.